examples/app.rb

```diff
@@ -4,7 +4,7 @@
 require "sinatra/base"
 
 class App < Sinatra::Base
-  SCHEMA = MultiJson.decode(File.read("schema.json"))
+  SCHEMA = JSON.parse(File.read("schema.json"))
```

Comment:
I suppose this one could be YAML load as well? Seems fine either way. Might be good to at least call out in the README or something that either/or is acceptable as whenever I bring that up around prmd people seem surprised.